demo/src/App.js

```diff
@@ -194,6 +210,19 @@ function App() {
             {colormapOn ? 'Colors' : COLORMAP}
           </Button>
         </div>
+        {loader && loader.isPyramid ? (
+          <div style={{ marginTop: '15px', marginBottom: '15px' }}>
+            <Button
+              variant="contained"
+              onClick={() => toggleOverview()}
+              key="overview"
+            >
+              {overviewOn ? 'Remove Overview' : 'Show Overview'}
+            </Button>
+          </div>
+        ) : (
+          []
+        )}
```

Comment:
For some reason when I toggle on and off the layer I get an error. Could you investigate this?

Reply:
If the error is in App.js, I don't really mind (but if its within `VivViewer` we should sort it out before merging).

Reply:
Resolved